deepchem/models/torch_models/ferminet.py

```diff
@@ -181,41 +250,29 @@ def __init__(
             raise ValueError("Given spin is not feasible")
 
         nucl = torch.from_numpy(self.nucleon_pos)
-        model = Ferminet(nucl,
-                         spin=(self.up_spin, self.down_spin),
-                         nuclear_charge=torch.tensor(charge),
-                         inter_atom=torch.tensor(
-                             compute_pairwise_distances(self.nucleon_pos,
-                                                        self.nucleon_pos)))
+        self.model = Ferminet(nucl,
+                              spin=(self.up_spin, self.down_spin),
+                              nuclear_charge=torch.Tensor(charge),
+                              batch_size=self.batch_no)
 
         self.molecule: ElectronSampler = ElectronSampler(
             batch_no=self.batch_no,
             central_value=self.nucleon_pos,
             seed=self.seed,
             f=lambda x: test_f(x),  # Will be replaced in successive PR
-            steps=1000,
-            steps_per_update=20
+            steps=self.random_walk_steps,
+            steps_per_update=self.steps_per_update
         )  # sample the electrons using the electron sampler
         self.molecule.gauss_initialize_position(
             self.electron_no)  # initialize the position of the electrons
-        adam = optimizers.AdamW()
+        self.prepare_hf_solution()
         super(FerminetModel, self).__init__(
-            model, optimizer=adam,
-            loss=L2Loss())  # will update the loss in successive PR
+            self.model,
+            loss=torch.nn.MSELoss())  # will update the loss in successive PR
 
-    def prepare_hf_solution(self, x: np.ndarray) -> np.ndarray:
```

Comment:
Why are we removing this argument?

Reply:
I am splitting this function into 2 smaller functions. This function here initializes the hartree fock pyscf fucntion (which can be called only once at start), and the other parameters will be part of the function which will be added in the next PR (this function will be called whenever electrons are sampled).